### PR TITLE
feat: Add --major-version-zero

### PR DESCRIPTION
**Pull request by sandzhaj**
Change default behavior for major version bumps.
Now, major versions are always bumped for breaking changes. --major-version-zero flag restores previous behaviour

Resolves: pawamoy#57
Issue #57: #57

### CODE REVIEW

File: src/git_changelog/cli.py [hunk 1]
```diff
@@ -153,7 +154,7 @@ def get_parser() -> argparse.ArgumentParser:
         dest="bump_latest",
         help="Deprecated, use --bump=auto instead. "
         "Guess the new latest version by bumping the previous one based on the set of unreleased commits. "
-        "For example, if a commit contains breaking changes, bump the major number (or the minor number for 0.x versions). "
+        "For example, if a commit contains breaking changes, bump the major number. "
```
Comment (pawamoy) [inline src/git_changelog/cli.py:157]:
```suggestion
        "For example, if a commit contains breaking changes, bump the major number (or the minor number for 0.x versions). "
```

File: src/git_changelog/cli.py [hunk 2]
```diff
@@ -165,7 +166,7 @@ def get_parser() -> argparse.ArgumentParser:
         metavar="VERSION",
         help="Specify the bump from latest version for the set of unreleased commits. "
         "Can be one of 'auto', 'major', 'minor', 'patch' or a valid semver version (eg. 1.2.3). "
-        "With 'auto', if a commit contains breaking changes, bump the major number (or the minor number for 0.x versions), "
+        "With 'auto', if a commit contains breaking changes, bump the major number, "
```
Comment (pawamoy) [inline src/git_changelog/cli.py:169]:
```suggestion
        "With 'auto', if a commit contains breaking changes, bump the major number (or the minor number for 0.x versions), "
```

**Comment by pawamoy** [comment]
For some reason this PR included changes from #60, so I removed them.

Eventually I decided to keep the current behavior of maintaining zerover by default. CI users can set `zerover = true` in their config file, then change it to `false` and commit a breaking change when they are ready to bump to v1. Then they can keep the setting, or remove it, since it has no effect after v1 is released.

CLI users just have to force bump to 1.0.0 or use `--bump auto -Z` when they're ready.

**Review by pawamoy — APPROVED** [review]

**Comment by sandzhaj** [comment]
That's great, thanks

**Pull request merged**